### PR TITLE
Create CONTRIBUTING & GUIDELINES

### PR DESCRIPTION
**Pull request by xerus2000**
not sure whether STANDARDS is the right name, but recently I wondered about some of these things again, so I thought instead of just figuring them out I would also write them out for future reference.

I considered calling it CONTRIBUTING, but our README consists mainly of contribution instructions.

### CODE REVIEW

**Review by anarchuser — COMMENTED** [review]
Looks good; I'm just worried about whether or not people will find this.
Regarding the name, maybe "Guidelines" fits better?

**Comment by xerus2000** [comment]
yeah, I think we should link it from the README, just wasn't sure where

**Comment by anarchuser** [comment]
maybe the [GitHub page regarding community health](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-default-community-health-file) helps you

**tl;dr** it suggests a CONTRIBUTING.md file in `root` or `docs/`

**Comment by xerus2000** [comment]
the issue with that is that the contributing instructions are already part of the readme - after all, what else should it contain?

**Comment by xerus2000** [comment]
- the intellij idea section is outdated (importing mechanism has changed)
- rename to GUIDELINES, that seems appropriate

**Review by SKoschnicke — APPROVED** [review]

**Comment by xerus2000** [comment]
I decided to move most stuff from the README to a new CONTRIBUTING. This is semantically correct and has the added advantage that we don't confuse pupils that encounter this repo and sometimes assumed they'd have to build the project ^^

We could then also put some more usage documentation into the README.

File: README.md [hunk 1]
```diff
@@ -4,81 +4,14 @@ Dies ist das offizielle Repository der [Software-Challenge](https://www.software
...
-  git clone git@github.com:CAU-Kiel-Tech-Inf/backend.git --recurse-submodules --shallow-submodules
-  ```
-- In IntelliJ: "File" > "New" > "Project from existing sources" > Projektverzeichnis auswählen
-  - "Import project from external model" > "Gradle" auswählen
-  - Im folgenden Fenster:
-    - "Use auto-import" ankreuzen
-    - bei "Gradle JVM" JDK 8 auswählen, wenn sie nicht schon ausgewählt ist
-    - "Finish" drücken
-- Warten, bis das Gradle-Build fertig ist
+Mehr Informationen zu Code-Strukturen und Standards findet ihr in den [GUIDELINES.md].
```
Comment (anarchuser) [inline README.md:15]:
```suggestion
Mehr Informationen zu Code-Strukturen und Standards findet ihr in den [GUIDELINES](https://github.com/CAU-Kiel-Tech-Inf/backend/blob/docs/standards/GUIDELINES.md).
```

**Comment by SKoschnicke** [comment]
Shouldn't everything pointed to developers of backend/gui (NOT pupils) be written in english?

File: README.md [hunk 2]
```diff
@@ -4,81 +4,14 @@ Dies ist das offizielle Repository der [Software-Challenge](https://www.software
...
 
-Nun können Gradle-Tasks auch direkt in IntelliJ vom Gradle-Tool-Fenster ausgeführt werden; dieses befindet sich normalerweise in der rechten Andockleiste.
+Wenn ihr mithelfen wollt, schaut in die [CONTRIBUTING.md].
```
Comment (anarchuser) [inline README.md:17]:
```suggestion
Wenn ihr mithelfen wollt, schaut in [CONTRIBUTING.md](https://github.com/CAU-Kiel-Tech-Inf/backend/blob/docs/standards/CONTRIBUTING.md).
```

**Review by anarchuser — COMMENTED** [review]
I think it's already a lot clearer this way

I'd just add corresponding links:

**Comment by xerus2000** [comment]
never use absolute links if you can avoid it!
They will break if you rename the repo, organisation or switch away the repo and point to the incorrect file if there's a fork.

I thought Github automatically linked file names, but I guess that was only true for full URLs.

**Comment by xerus2000** [comment]
> Shouldn't everything pointed to developers of backend/gui (NOT pupils) be written in english?

yeah, but it's not worth to rewrite it just for the sake of it. I'd do that incrementally if there are major changes.
